src/ol_infrastructure/applications/codejail/Pulumi.applications.codejail.mitxonline.Production.yaml

```diff
@@ -4,8 +4,8 @@ encryptedkey: AQICAHg/+QzF9hGIaoayDitgnEVHEhuaANONVQQOnqpkIsol1gFMzAtzjNTziE5CH3
 config:
   aws:region: us-east-1
   codejail:auto_scale:
-    desired: 3
-    max: 10
+    desired: 2
```

Comment:
We should probably keep this at 3 for a minimum as well to ensure we have enough capacity if it's needed.